README.rst

```diff
@@ -335,6 +336,30 @@ The following options can be only specified in the task decorator:
   has set up ``BATCH_QUEUES`` for the specific queue (see the *Configuration*
   section).
 
+- ``schedule``
+
+  If given, makes a task execute periodically. Pass either:
+
+  - a function that takes the current datetime as an argument.
+  - a tuple ``(f, args)``, where ``f`` takes the current datetime as the first
+    argument, followed by the additional args.
+
+  The schedule function must return the next task execution datetime, or
+  ``None`` to prevent periodic execution. The function is executed to determine
+  the initial taks execution date when a worker is initialized, and to determine
+  the next execution date when the task is about to get executed.
+
+  For most common scenarios, the ``periodic`` built-in function can be passed:
+
+  - ``periodic(seconds=0, minutes=0, hours=0, days=0, weeks=0, start_date=None,
```

Comment:
For larger intervals (days, weeks) it might be useful to have a flag like `run_on_weekends=True` and specify time of the day to start execution. For example, someone wants to run some cleanup on only on weekdays at 3am.